test/utils.test.js

```diff
@@ -921,3 +921,26 @@ describe('utils.parseContentTypeFromResponse', () => {
         expect(parsedReallyBad.charset).toBe('utf-8');
     });
 });
+
+describe('utils.purgeLocalStorage()', () => {
+    // Create test folder
+    const folder = Date.now().toString(16);
+    fs.mkdirSync(folder);
+
+    test('should purge local storage by default', () => {
+        expect(() => utils.purgeLocalStorage()).not.toThrowError();
```

Comment:
the method is async, we should test it via `expect.resolves/rejects` instead

```suggestion
        await expect(utils.purgeLocalStorage()).resolves.toBeUndefined();
```